src/@ionic-native/plugins/appsflyer/index.ts

```diff
@@ -95,7 +95,7 @@ export class Appsflyer extends IonicNativePlugin {
    * @param {AppsflyerEvent} eventValues event details
    */
   @Cordova({ sync: true })
```

Comment:
This function shouldn't be sync

https://github.com/AppsFlyerSDK/appsflyer-cordova-plugin/blob/master/www/appsflyer.js#L128-L131